frontend/src/pages/ProcessDataPage/ProcessedDataPage.tsx

```diff
@@ -22,65 +28,296 @@ const ProcessedDataPage = function () {
   const rollbar = new Rollbar(rollbarConfig);
   rollbar.debug("Reached Processed Data page");
 
+  const [files, setFiles] = useState<any[]>([]);
+  const [currentFile, setCurrentFile] = useState<any>();
+  const [selectedModel, setSelectedModel] = useState<string>("svm");
+
+  const fitBitService = new FitBitService();
+  const appleWatchService = new AppleWatchService();
```

Comment:
As for calling our api calls in the fronend. I suggest going the approach of react hooks. Why? It is because itll make our lives way easier for testing and a more cleaner and readable way of handling data.

Take a look at LoginForm on how I used our login function for calling stuff.

We also have some hooks living in shared/hooks on the structure of how I tested those as well.